components/style/gecko_bindings/sugar/ns_t_array.rs

```diff
@@ -93,4 +93,13 @@ impl<T> nsTArray<T> {
         let mut header = self.header_mut();
         header.mLength = len;
     }
+
+    /// Resizes an array containing only POD elements
+    ///
+    /// This is safe since uninitialized C-obtained POD elements are fine
+    pub fn set_len_pod(&mut self, len: u32) {
```

Comment:
Yeah, the assumption here is that the only types being used are types from C. But perhaps this should be unsafe too.